src/Glpi/Form/Clone/FormCloneHelper.php

```diff
@@ -365,7 +380,15 @@ private function updateUuidsInConditionDataJson(string $json): string
 
             // Get correct uuid
             $type = Type::from($raw_type);
-            $new_uuid = $this->getMappedUuidForConditionItem($type, $uuid);
+            try {
+                $new_uuid = $this->getMappedUuidForConditionItem($type, $uuid);
+            } catch (MappedItemNotFoundException $e) {
+                // We try to prevent invalid conditions from happening on forms
+                // but if it happens we prefer to ignore them rather than block
```

Comment:
Maybe an entry could be added in logs to inform user there is something wrong?

Reply:
I thought about it but there is nothing to be done for the user, the condition is not visible for the user and it will clean itself the next time the form is saved.

So I think we can keep it like that.